test/localtest.sh

```diff
@@ -206,19 +206,19 @@ fi
 
 if [ -n "$DD_APM_INSTRUMENTATION_LANGUAGES" ]; then
   if [[ "$OS_TYPE" == "ubuntu" ]]; then
-    debsums -c datadog-apm-library-dotnet
-    debsums -c datadog-apm-library-java
-    debsums -c datadog-apm-library-js
-    debsums -c datadog-apm-library-python
-    debsums -c datadog-apm-library-ruby
-    echo "[OK] Inject libraries installed"
+      ls /opt/datadog-packages/datadog-apm-library-dotnet || debsums -c datadog-apm-library-dotnet
+      ls /opt/datadog-packages/datadog-apm-library-java || debsums -c datadog-apm-library-java
+      ls /opt/datadog-packages/datadog-apm-library-js || debsums -c datadog-apm-library-js
+      ls /opt/datadog-packages/datadog-apm-library-python || debsums -c datadog-apm-library-python
+      ls /opt/datadog-packages/datadog-apm-library-ruby || debsums -c datadog-apm-library-ruby
+      echo "[OK] Inject libraries installed"
   else
-    rpm --verify --nomode --nouser --nogroup datadog-apm-library-dotnet
-    rpm --verify --nomode --nouser --nogroup datadog-apm-library-java
-    rpm --verify --nomode --nouser --nogroup datadog-apm-library-js
-    rpm --verify --nomode --nouser --nogroup datadog-apm-library-python
-    rpm --verify --nomode --nouser --nogroup datadog-apm-library-ruby
-    echo "[OK] Inject libraries installed"
+      ls /opt/datadog-packages/datadog-apm-library-dotnet || rpm --verify --nomode --nouser --nogroup datadog-apm-library-dotnet
+      ls /opt/datadog-packages/datadog-apm-library-java || rpm --verify --nomode --nouser --nogroup datadog-apm-library-java
+      ls /opt/datadog-packages/datadog-apm-library-js || rpm --verify --nomode --nouser --nogroup datadog-apm-library-js
+      ls /opt/datadog-packages/datadog-apm-library-python || rpm --verify --nomode --nouser --nogroup datadog-apm-library-python
+      ls /opt/datadog-packages/datadog-apm-library-ruby || rpm --verify --nomode --nouser --nogroup datadog-apm-library-ruby
+      echo "[OK] Inject libraries installed"
```

Comment:
I see and appreciate a proud member of the 4 spaces as tabs team :saluting_face: